lib/glimesh_web/live/user_settings/components/profile_settings_live.html.leex

```diff
@@ -150,15 +150,20 @@
 
                             <%= textarea f, :profile_content_md, [class: "form-control mb-4 text-monospace", id: "textarea_edit", rows: 20] %>
                         </div>
-                        <div class="tab-pane form-control mb-4" id="preview">
-                            <p><%= gettext("Coming soon!") %></p>
+                        <div class="tab-pane form-control mb-4" style="height:auto" id="preview">
+                            <%= if @user.profile_content_html do %>
```

Comment:
From the looks of this, it is not a live preview but instead of a rendering of the user's previous profile content. Am I reading it right?

Reply:
Yeah, if you were to open a new tab to edit and preview it you wouldn't see the changes. I assumed that would be fine since the only place you can edit the profile was on that page, as long as you save the changes they will appear. I can try to make it appear without needing to save it.

Reply:
Yeah, the intention of the Preview tab is to be a live preview of your profile to make editing easier.